pkg/operator/ceph/cluster/cluster.go

```diff
@@ -96,12 +98,13 @@ func (c *cluster) doOrchestration(rookImage string, cephVersion cephver.CephVers
 	}
 
 	// Start the mon pods
+	opcontroller.UpdateCondition(c.context, c.namespacedName, cephv1.ConditionProgressing, v1.ConditionTrue, opcontroller.ClusterProgressingReason, "Reconciling mons")
```

Comment:
```suggestion
	opcontroller.UpdateCondition(c.context, c.namespacedName, cephv1.ConditionProgressing, v1.ConditionTrue, opcontroller.ClusterProgressingReason, "Reconciling mon(s)")
```

Reply:
I wonder if "reconciling" is meaningful, it feels really attached to the controller/operator patter where I think we are trying to indicate progress in the CRD.

What about "Configuring Ceph $COMPONENT" instead? IMO it's clearer for users to understand what's going on.

Reply:
Agreed, "Configuring Ceph $COMPONENT" sounds better